### ISSUE TITLE
Instructions for how to contribute to the website-wiki repo

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4364]
### Dependency
- [x] #3636 
- [ ] We are ready for the website-wiki repo to accept contributions (edits, additions, deletions, etc)

### Overview
We need instructions for how to contribute to the hackforla/website-wiki repo so that team members can contribute to our documentation.

### Action Items
- [ ] Determine the best place to add these instructions: CONTRIBUTING.md or on the website-wiki repo. If in the website-wiki repo, please confer with the Technical Writer on the best location for these instructions. 
- [ ] Write instructions for how to contribute to the hackforla/website-wiki repo

### Resources/Instructions
- [hackforla/website-wiki](https://github.com/hackforla/website-wiki)

**Comment by JessicaLucindaCheng** [comment]
@jdingeman I wasn't sure what your plan was for the roll out of contributing to the website-wiki repo. I wrote this up because I have some wiki editing/adding issues that would need this as a dependency (also any other dependencies that you think are needed before we start allowing contributions into the website-wiki repo). 

Please edit the issue as needed for `ready for prioritization`.

**Comment by jdingeman** [comment]
Dev leads need to understand what the plan for introducing the website-wiki is and what kinds of automations, branches, issues, etc will go into it.

**Comment by JessicaLucindaCheng** [comment]
@roslynwythe
- Is this a duplicate of https://github.com/hackforla/website/issues/6142? If yes, decide on which duplicate issue you want to close.
